GameOfRunes/GameOfRunes/Utils/GameConfig.swift

```diff
@@ -15,6 +15,10 @@ import UIKit
 enum GameConfig {
     static let fontName = "DragonFire"
     
+    enum AnimationNodeKey {
+        static let enemy_walking = "enemy-walking-animation"
```

Comment:
Following up on previous comment in `MoveSystem`, should use this as the `animationNodeKey` that is passed into `stopAnimation`